server/sonar-server/src/main/java/org/sonar/server/issue/Action.java

```diff
@@ -78,6 +78,8 @@ public boolean supports(DefaultIssue issue) {
     DefaultIssue issue();
 
     IssueChangeContext issueChangeContext();
+
+    String issueOrganizationUuid();
```

Comment:
remark: it's surprising to add `issueOrganizationUuid` as the signature of this high level Action. Should it be part of `DefaultIssue` for example?